docs/source/design/composite-time-series.rst

```diff
@@ -0,0 +1,220 @@
+            "notes": "text",
+            "version", "version date", // maybe not? could just use POR or period-of-record in the ts id version
+            // if values that equals the start or end timestamp are included
+            "start-inclusive": true,
+            "end-inclusive": false
+            // suggest default of "start-inclusive": true, "end-inclusive": false
+            // it may also make sense to just make this *always* the above and not let the user set it.
+            // alternatively if this is always [) then only start is required.
+            // however, the class would required and end field as the actual Time Series output needs to know the actual end.
+        }
+    ]
+    // array above *should* be sorted by start when provided to user.
+    }
+
+
+Operations required:
+
+* Create
+* Remove member (ts id + range)
+* Add member
+* List members
+* Replace all members?
+* Delete
+
+
+Composite Time Series Response
+==============================
+
+.. code-block::jsonc
+
+    {
+    // ... as current TimeSeries JSON
+    "composite-members-present": [
+        // member definition from above
+    ] 
+    }
+
+
+Supported Operations:
+
+* Get, through existing TimeSeries classes.
+
+Does it make sense to support writing directly to a composite time series. While the write of each element *could* be sent to the underlying member, this seems ripe for error when editing or updating any data. It is likely that any edits would always be to the most recent time series, and configured in some other system.
+
+Storage of member information
+================================
+
+#. Store in Clob as we refine the design
+#. Create appropriate tables once the design is stable.
+
+System responsibility for "knowing" to process composite.
+=========================================================
+
+Time Series Catalog
+-------------------
+
+Time Series Catalog should show composite time series and allow searching by "authoritative"
+
+TimeSeries DTO
+--------------
+
+Add nullable "members" property.
+
+TimeSeriesDao
+-------------
+
+If the system sees the "Composite" marker retrieve the members for the range and build the time series.
+
+NOTE: considering the user may request the *entire* Period-of-record, this is a good opportunity to see that, start the retrieval in a job queue, and return a status URL to the user for future download. I have see such mechanism for bulk data in other systems. Maybe return an "I'm working on it variant" that the controller can know how to format.
+
+Error handling and other conditions.
+====================================
+
+Versioned (date) time series
+----------------------------
+
+As the composite time series is comprised of multiple other time series should this always be an error to specify?
+The marker for always latest or always first may make sense to allow, however, at the time series is supposed to be authoritative, that would add ambiguity.
```

Comment:
You have to define what "authoritative" means in order to properly answer this question.